lib/gooddata/models/user_filters/user_filter_builder.rb

```diff
@@ -294,8 +338,7 @@ def self.maqlify_filters(filters, project_users, options = {})
         expression, errors = create_expression(f, labels_cache, lookups_cache, attrs_cache, options)
         safe_login = login.downcase
         profiles_uri = if options[:type] == :muf
-                         project_user = project_users.find { |u| u.login == safe_login }
-                         project_user.nil? ? ('/gdc/account/profile/' + safe_login) : project_user.profile_url
+                         user_profile_mapping[safe_login]
```

Comment:
what happened if user not in the domain.users, we should log as error and skip that user.

Reply:
Changed to use the login name if not found, thanks